tests/resources/Nimbus-Util.robot

```diff
@@ -433,6 +433,34 @@ Deploy Simple NFS Testbed
 
     [Return]  ${user}-${name}.nfs.0  ${user}-${name}.nfs.1  ${user}-${name}.esx.0  ${nfs-ip}  ${nfs-ro-ip}  ${esx-ip}
 
+Deploy Simple VC Cluster With NFS Testbed
+    [Arguments]  ${user}  ${password}  ${additional-args}=
+    ${name}=  Evaluate  'NFS-' + str(random.randint(1000,9999)) + str(time.clock())  modules=random,time
+    Log To Console  \nDeploying Nimbus NFS testbed: ${name}
+    Open Connection  %{NIMBUS_GW}
+    Wait Until Keyword Succeeds  2 min  30 sec  Login  ${user}  ${password}
+
+    ${out}=  Execute Command  ${NIMBUS_LOCATION} nimbus-testbeddeploy --testbedName nfs --runName ${name} ${additional-args}
+    Log  ${out}
+    # Make sure the NFS deploy worked
+    Should Contain  ${out}  ${name}.nfs.0' is up. IP:
+    # Make sure the VC deploy worked
+    Should Contain  ${out}  .vc.0' is up. IP:
```

Comment:
just look for DEPLOYMENT => PASS or whatever.  don't look for these specific things